app/renderer/components/preferences/payment/ledgerTable.js

```diff
@@ -350,11 +357,18 @@ const styles = StyleSheet.create({
   },
 
   verifiedTd: {
-    padding: '0 0 0 15px'
+    // Set the same padding with the padding-top and padding-bottom
+    paddingRight: '.3ch !important',
```

Comment:
I like using `ch` units for width/height but for padding seems obscure. Let me know your thoughts on why this

Reply:
it sounds fair. I will revisit the padding values of sortable table to see which one works most properly.